tests/integration/security/authorization_test.go

```diff
@@ -618,169 +618,156 @@ func TestAuthorization_EgressGateway(t *testing.T) {
 	framework.NewTest(t).
 		Features("security.authorization.egress-gateway").
 		Run(func(t framework.TestContext) {
-			ns := namespace.NewOrFail(t, t, namespace.Config{
-				Prefix: "v1beta1-egress-gateway",
-				Inject: true,
-			})
+			ns := apps.Namespace1
```

Comment:
Similar issue here, see the comment in ingress test. The egress test is really to test the egress gateway itself, authz polices are applied to egress directly. It does not matter too much whether you are sending the requests from k8s or vm, they do not have authz policies at all.

Unless you can run the egress gateway in VM, I think there is no much benefit to change the source/dst workload to vm.

Reply:
We will support gateways on VM, the purpose of this change is to cover the communication between gateways and services on VM, and verify that users can still apply policies on gateways to control the access to VM services.

Reply:
can you add a comment for this?

Reply:
Added.